wire-ios/Wire-iOS/Sources/UserInterface/Helpers/UIFont+FontSpec.swift

```diff
@@ -75,6 +88,14 @@ extension UIFont {
         return FontSpec(.normal, .semibold).font!
     }
 
+    class var headerSemiBoldFont: UIFont {
+        return FontSpec(.header, .semibold).font!
+    }
+
+    class var mediumRegular: UIFont {
+        return FontSpec(.medium, .regular).font!
```

Comment:
I believe we should use `FontSpec` instead of `UIFont`. did @agisilaos show it in the presentation about accessibility? However I don't know if it works with SwiftUI

Reply:
We have both of these fonts in FontSpec

Reply:
We can't use it that way. We could only use `UIFont`s converted to `Font`

Reply:
Replacing them

Reply:
done